### PR TITLE
Add ``display_name`` field to ``Vulnerability``

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/security/models.py [hunk 1]
```diff
@@ -46,7 +46,13 @@ class Vulnerability(
     name = models.CharField(
         verbose_name=_("name"),
         max_length=1024,
-        unique=False)
+        unique=False
+    )
+    short_name = models.CharField(
+        verbose_name=_("short name"),
+        max_length=1024,
+        unique=False
```
Comment (romcheg) [inline src/ralph/security/models.py]: I would say it should be unique -- otherwise that means the same vulnarability

Comment (szok) [inline src/ralph/security/models.py]: delete unique=false

**Review by romcheg — CHANGES_REQUESTED** [review]
One nit inline. Otherwise LGTM

**Review by szok — DISMISSED** [review]

File: src/ralph/security/models.py [hunk 2]
```diff
@@ -46,7 +46,12 @@ class Vulnerability(
     name = models.CharField(
         verbose_name=_("name"),
         max_length=1024,
-        unique=False)
+        unique=False
+    )
+    short_name = models.CharField(
+        verbose_name=_("short name"),
+        max_length=1024
```
Comment (romcheg) [inline src/ralph/security/models.py]: From what I see unique is False by default: https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L135

**Review by romcheg — CHANGES_REQUESTED** [review]

File: src/ralph/security/models.py (hunk 2 above)
Comment (ar4s) [reply src/ralph/security/models.py]: I renamed this field to ``display_name``

**Review by romcheg — DISMISSED** [review]
All my concerns have been resolved. LGTM, thanks!

**Review by szok — APPROVED** [review]

**Pull request merged**